scripts/lochness_create_template.py

```diff
@@ -137,40 +138,85 @@ def create_keyring_template(keyring_loc: Path, args: object) -> None:
     template_dict = {}
     template_dict['lochness'] = {}
 
-    if 'redcap' in args.sources:
+    if 'redcap' in args.sources or 'upenn' in args.sources:
+        if args.enter_passwords:
+            url = getpass.getpass('REDCAP URL: ')
+            api_token = getpass.getpass('REDCAP API_TOKEN: ')
+            upenn_url = getpass.getpass('UPENN REDCAP URL: ')
+            upenn_api_token = getpass.getpass('UPENN API_TOKEN: ')
+        else:
+            url = 'https://redcap.partners.org/redcap'
+            api_token = '*****'
+            upenn_url = 'https://redcap.upenn.org/redcap'
+            upenn_api_token = '*****'
+
         template_dict['lochness']['REDCAP'] = {}
         template_dict['lochness']['SECRETS'] = {}
         for study in args.studies:
-            study_dict = {f'redcap.{study}': [study]}
+            project_name = study[:-2]
+            if 'upenn' in args.sources and 'redcap' in args.sources:
+                study_dict = {f'redcap.{project_name}': [project_name],
+                              'redcap.UPENN': ['UPENN']}
+                template_dict['redcap.UPENN'] = {
```

Comment:
Since `study_dict` has two {key,value} pairs, is it right that `template_dict` has only `redcap.UPENN`? My doubt is enhanced by the identity of `template_dict` with that in the `elif` block.

Reply:
I think you got it. It's expecting something like below.

![Pasted_Image_9_16_21__11_40_AM](https://user-images.githubusercontent.com/3075783/133642789-d588c335-5884-4338-8cbc-91d53f40caf7.png)